internal/http/services/archiver/archiver.go

```diff
@@ -105,38 +106,73 @@ func (c *Config) init() {
 	c.GatewaySvc = sharedconf.GetGatewaySVC(c.GatewaySvc)
 }
 
+func (s *svc) getFiles(ctx context.Context, files, ids []string) ([]string, error) {
+	if len(files) == 0 && len(ids) == 0 {
+		return nil, errtypes.BadRequest("file and id lists are both empty")
+	}
+
+	var f []string
+
+	for _, id := range ids {
+		// an id must have the form <storage_id>:<id>
+
+		split := strings.Split(id, ":")
```

Comment:
The resource ids arrive in the web ui as base64 encoded strings. With this implementation I'd need to decode the fileId in the web ui and send it in decoded form. Just knowing this I'd prefer if you could the base64 decoding here instead. But I'm not sure if I'm seeing the full picture here. Please let me know if the decoding should happen in the clients or here.

Reply:
Thanks for the review, fixed to receive a base64 encoded resource ID to be consistent with the rest of the backend.